src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -366,9 +451,51 @@ class CreateMafVcf : CliktCommand(help = "Create g.vcf and h.vcf files from Anch
                 else lastVariant.getAttributeAsInt("ASM_Start",region.second.position)
         }
 
-        val regions = buildNewAssemblyRegions(newASMStart,newASMEnd,variants)
+        // Check if asm region size is >= 5% of reference region size
```

Comment:
I thought we decided to not do this yesterday at programmers meeting.  Making reference ranges have a minimum size makes sense, but we can have small or even missing haplotypes

Reply:
yes we did.  This was a branch I worked on some time ago and went back to yesterday.  I forgot to remove the assembly code - will do that now.